client/me/security-2fa-backup-codes/index.jsx

```diff
@@ -139,7 +139,7 @@ class Security2faBackupCodes extends React.Component {
 	render() {
 		return (
 			<div className="security-2fa-backup-codes">
-				<SectionHeader label={ this.props.translate( 'Backup Codes' ) }>
+				<SectionHeader label={ this.props.translate( 'Backup codes' ) }>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.